docs/workshop/model_deployment.md

```diff
@@ -1,8 +1,57 @@
 # Model Deployment
 
-Deploy your trained model
-* Instructions on how to create an image with prediction script and requirements (only documentation, not being done live at workshop)
-* Create a deployment using Seldon operator.
-* Create a route to the Seldon service.
-* Interact with an already trained model from Jupyeterlab.
-* Section where users can directly access and interact with the service using the pre-trained model and existing URL or the newly created route to the model.
+Once you complete the process of feature engineering and model training and come up with the best possible model. It is now time to deploy your model as a service. We use [Seldon](https://docs.seldon.io/projects/seldon-core/en/latest/wrappers/s2i.html) Operator on Red Hat OpenShift in order to create an endpoint which can be accessed from a terminal or jupyter notebook or can even be integrated with a bot which can directly comment on your new PullRequest.
+
+In order to deploy your model,
+
+1. Fork the repo containing the [deployment config file](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/notebooks/time-to-merge-prediction/thoth-station/thoth-ttm-deployment-config.yaml). Once you fork it, change the model path and deployment name in the config file and push it to your origin repo. You will be using this config file to create Seldon Deployment later.
```

Comment:
Do they need to fork the repo just to deploy the model? Should we assume they are already working on their own fork at this point?

Reply:
yeah, they should be already working off a fork at this point.

Reply:
Thanks guys. I have updated the file by removing the line from the document. Assuming that the instruction will be there is other instruction file.